### PR TITLE
Update ve.m3u

### PR DESCRIPTION
**Pull request by Carlinhos027**

### CODE REVIEW

File: streams/ve.m3u [hunk 1]
```diff
@@ -116,3 +116,11 @@ https://ls.tves.gob.ve/hls/tves.m3u8
 https://vcp3.myplaytv.com/trv/trv/playlist.m3u8
 #EXTINF:-1 tvg-id="CanalCulturaVenezuela.ve",Canal Cultura Venezuela (1080p) [Not 24/7]
 https://ythls.onrender.com/channel/UCMPPdsYmf-V6jukDtIfpM_A.m3u8
+#EXTINF:-1 tvg-id="GlowTV.ve",Glow TV (720p) [Not 24/7]
+https://vcp7.myplaytv.com/glowtv/glowtv/playlist.m3u8
+#EXTINF:-1 tvg-id="CanalDiplomatico.ve",Canal Diplomático (720p) [Not 24/7]
+https://vcp.myplaytv.com/canaldiplomatico/canaldiplomatico/playlist.m3u8
+#EXTINF:-1 tvg-id="SomosTV.ve",Somos TV (720p)
```
Comment (BellezaEmporium) [inline streams/ve.m3u:123]:
```suggestion
#EXTINF:-1 tvg-id="SomosTV.ve",Somos TV (720p) [Not 24/7]
```

**Review by BellezaEmporium — DISMISSED** [review]
LGTM. Will check as vcp.myplaytv.com doesn't seem to answer, however the rest seem to. I do not know if there's a "master server" that does load balancing with the other vcp servers.

**Comment by Carlinhos027** [comment]
> LGTM. Will check as vcp.myplaytv.com doesn't seem to answer, however the rest seem to. I do not know if there's a "master server" that does load balancing with the other vcp servers.

https://vcp7.myplaytv.com/canaldiplomatico/canaldiplomatico/playlist.m3u8

That's the right link. Will fix that.

**Review by Carlinhos027 — COMMENTED** [review]
Doesn't work. 404.

**Comment by BellezaEmporium** [comment]
> Doesn't work. 404.

Talking about the new TLT ? Even the original one 404s. I guess it's not 24/7 too.

**Comment by Carlinhos027** [comment]
> > Doesn't work. 404.
> 
> Talking about the new TLT ? Even the original one 404s. I guess it's not 24/7 too.
Official page broadcasts on Dailymotion so idk:
https://teletuya.net/en-vivo/

**Comment by BellezaEmporium** [comment]
> > > Doesn't work. 404.
> > 
> > 
> > Talking about the new TLT ? Even the original one 404s. I guess it's not 24/7 too.
> 
> Official page broadcasts on Dailymotion so idk: https://teletuya.net/en-vivo/

https://stream-01.dc3.dailymotion.com/sec(_i51VaJAaJ6AK3dDJPgpsXA9jkGCr7Q77plpQXkEMng_JPfXc9XMSsvxRzZ_Jjrt)/dm/3/x8my9op/s/live-0.m3u8#cell=lcore

**Review by freearhey — APPROVED** [review]

**Review by UltraHDR — APPROVED** [review]

**Pull request merged**